librz/core/cmd/cmd_api.c

```diff
@@ -1170,13 +1179,13 @@ static void fill_argv_modes_help_strbuf(RzCmd *cmd, RzStrBuf *sb, RzCmdDesc *cd,
 	if (max_len - min_len > MAX_RIGHT_ALIGHNMENT) {
 		max_len = min_len + MAX_RIGHT_ALIGHNMENT;
 	}
-
+	bool scr_utf8 = cmd->core ? cmd->core->config ? rz_config_get_b(cmd->core->config, "scr.utf8") : false : false;
 	size_t i;
 	for (i = 0; i < RZ_ARRAY_SIZE(argv_modes); i++) {
 		if (cd->d.argv_modes_data.modes & argv_modes[i].mode) {
 			char *name = rz_str_newf("%s%s", cd->name, argv_modes[i].suffix);
 			char *summary = rz_str_newf("%s%s", cd->help->summary, argv_modes[i].summary_suffix);
-			do_print_child_help(cmd, sb, cd, name, summary, false, max_len, use_color);
+			do_print_child_help(cmd, sb, cd, name, summary, scr_utf8 ? RUNE_LINE_VERT : "| ", false, max_len, use_color);
```

Comment:
You are doing this within a for loop, define `vertical_line` outside of the loop and then pass it as variable here.